.circleci/config/jobs/test-ui.yml

```diff
@@ -9,19 +9,8 @@ steps:
   - run:
       name: Test UI
       command: |
-        set -x
-
-        # Install Chrome
-        wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub \
-          | apt-key add -
-        echo "deb http://dl.google.com/linux/chrome/deb/ stable main" \
-          | tee /etc/apt/sources.list.d/google-chrome.list
-        apt-get update
-        apt-get -y install google-chrome-stable
-        rm /etc/apt/sources.list.d/google-chrome.list
-        rm -rf /var/lib/apt/lists/* /var/cache/apt/*
-
         # Add ./bin to the PATH so vault binary can be run by Ember tests
+        set -x
```

Comment:
In general I avoid using `set -x` in CI, because it has a tendency to leak secrets into the logs (if not immediately, then later on when someone makes further changes). It's fine for a quick debugging session as long as you're careful, but I tend to avoid it completely and just add manual logging, e.g. `echo "SOMEVAR=$SOMEVAR"` or `echo "this conditional was hit!" or whatever, when I'm debugging.

Reply:
I carried this over from our previous testing set up, because it failed and didn't seem to set the path properly on the following line `export PATH="${PWD}/bin:${PATH}"` and I wasn't sure why. 

I read about avoiding use, so I'd prefer to remove it if there's a better way!